src/rpc/server.cpp

```diff
@@ -39,6 +39,24 @@ static struct CRPCSignals
     boost::signals2::signal<void (const CRPCCommand&)> PreCommand;
 } g_rpcSignals;
 
+std::string CRPCCommand::Label (const ERPCCategory theCategory){
+    static const char * kVect [] ={
+        "blockchain",
+        "control",
+        "generating",
+        "hidden",
+        "mining",
+        "network",
+        "rawtransactions",
+        "util",
+        "wallet",
+        "zmq",
+        "test"
+    };
```

Comment:
This implementation means that the list here has to be in sync with the enum definition.  I think it would be much more robust and readable if you had here a big `switch` instead.

Reply:
Done https://github.com/bitcoin/bitcoin/pull/13945/commits/cf190f57a078df1d37bd1822e3fa69ba0d9c90e6